pages/Documents/ConversationalAI/Third-PartyBots/e-bot7/basic-content.md

```diff
@@ -0,0 +1,121 @@
+---
+pagename: Basic Content
+sitesection: Documents
+categoryname: "Conversational AI"
+documentname: Third-Party Bots
+subfoldername: e-bot7
+permalink: third-party-bots-e-bot7-basic-content.html
+indicator:
+---
+
+### The Welcome Event
+
+The behavior of the welcome event is different depending on whether the bot is for chat and messaging. This divergence comes down to the way that each individual LivePerson product works.
+
+#### Messaging Conversations
+
+A Messaging conversation qualifies as "initiated" from a Conversational Cloud perspective only after the consumer sends
+their first message. The consumer is prompted for their initial message in the channel they have chosen to initiate the
+conversation. As a result, the consumer’s first message will be the entry point for the main flow in e-bot7, which can be processed by a `Visitor Input` node.
+In this scenario, all bot messages in the flow which are created before this node will be skipped.
+
+<img class="fancyimage" style="width:450px" src="img/ThirdPartyBots/e-bot7-messaging-flow.png">
+<figure>Figure 2.1 Example conversation flow for messaging without support of the `Welcome` event</figure>
+<br>
```

Comment:
Please remove this and other (3) line breaks.

(If there is a styling issue, let @itay1313 and me know and we can advise—in the worst case we insert another line in the Markdown.)

Reply:
@j9t Removed all occurrences of br tags. But now there is no space between figures and their next lines. If I would add empty lines, they get killed by auto formatting which is as expected.

Problem here: figure tag doesn't refer to any visible style. I would expect to have the text inside centered (and italic) with smaller font-size with more margin to the bottom and less to the top.

<img width="978" alt="Screenshot 2022-07-04 at 09 52 59" src="https://user-images.githubusercontent.com/1105677/177109019-17812970-8b4b-4c35-8a4d-2c2f81088862.png">

Reply:
Let’s do the following, and let’s do this right:

Please switch to the following structure for all figures:

```html
<figure>
 <img src="ex-a.png" alt="Two squiggles on a dirty piece of paper.">
 <figcaption>Exhibit A. The alleged <cite>rough copy</cite> comic.</figcaption>
</figure>
```

I.e., in this particular case (simplified, check on need for classes):

```html
<figure>
 <img src="img/ThirdPartyBots/e-bot7-messaging-flow.png" alt="TODO">
 <figcaption>Figure 2.1: Example conversation flow for messaging without support of the `Welcome` event.</figcaption>
</figure>
```

For styling, @itay1313, could you help @cscharun if need be so that this matches/resembles our current design?

(As we’re just getting started with a new team on the dev site, let’s make adjustments where needed. For example, if it slows us down to coordinate on the design, and if looks largely okay, let’s merge, and Itay and I will make the design adjustments separately.)

Thanks all!

Reply:
Hey @j9t, updated as suggested. @itay1313 Please take care of the styles. I currently left the explicit specification of the width in the style attribute to keep images sharp.